File: Oxygene/Java/Android/com.remobjects.CardView/MainActivity.pas
namespace com.remobjects.cardview;

interface

uses
  java.util,
  android.app,
  android.content,
  android.os,
  android.util,
  android.view,
  android.widget;

type
  MainActivity = public class(Activity)
  public
    method onCreate(savedInstanceState: Bundle); override;
  end;

implementation

//Launcher Activity for the CardView sample app.

method MainActivity.onCreate(savedInstanceState: Bundle);
begin
  inherited;
  // Set our view from the "main" layout resource
  ContentView := R.layout.activity_card_view;

  if (savedInstanceState = nil) then 
  begin
    getFragmentManager().beginTransaction().add(R.id.container, CardViewFragment.newInstance()).commit()
  end;

end;

end.

File: Hydrogene/Java/Android/CardView/MainActivity.cs
using java.util;
using android.app;
using android.content;
using android.os;
using android.util;
using android.view;
using android.widget;

namespace com.remobjects.cardview
{
	public class MainActivity: Activity
	{
		public override void onCreate(Bundle savedInstanceState)
		{
			base.onCreate(savedInstanceState);

			// Set our view from the "main" layout resource
			ContentView = R.layout.activity_card_view;

      if (savedInstanceState == null)
      {
        getFragmentManager().beginTransaction().add(R.id.container, CardViewFragment.newInstance()).commit();
      }
		}
	}
}
